src/basic_bot/services/central_hub.py

```diff
@@ -43,7 +45,7 @@ def iseeu_message(websocket):
     )
 
 
-async def send_message(websocket, message):
+async def send_message(websocket: Union[WebSocketServerProtocol, str], message: str) -> None:
```

Comment:
The `Union` here looks suspect.  Why would `websocket` ever be a string?

Reply:
see comment reply below.  remove "all" possibility as it is no longer supported.